datashare-app/src/main/java/org/icij/datashare/com/bus/amqp/AmqpInterlocutor.java

```diff
@@ -32,8 +34,8 @@ public class AmqpInterlocutor {
 
 
     @Inject
-    public AmqpInterlocutor(PropertiesProvider propertiesProvider) throws IOException {
-        this(new Configuration(propertiesProvider.getProperties()));
+    public AmqpInterlocutor(PropertiesProvider propertiesProvider) throws IOException, URISyntaxException {
+        this(new Configuration(new URI(propertiesProvider.get("messageBusAddress").orElse("amqp://localhost:5672"))));
```

Comment:
I think most default values use then docker-compose hostnames (such as `redis://redis`, `http://elasticsearch:9200`, etc). Should we use the same here?

```suggestion
        this(new Configuration(new URI(propertiesProvider.get("messageBusAddress").orElse("amqp://amqp:5672"))));
```

Reply:
Right. 
Actually we've put the names of the services that we had in the development environment. For automated integration tests and for manual local testing. Here `localhost` is used because we use qpid for unit tests, not the docker container. 

If we'd want to follow the same rule, we should use the _broker_ name i.e. `amqp://rabbitmq:5672`.
What do you think?

Reply:
Yes, `rabbitmq` makes much more sense to me :)

Reply:
https://github.com/ICIJ/datashare/pull/1284/commits/3b582f0454a9059e0997ce89033bb706ef4e38da